### PR TITLE
Fix Streamsets chart

### PR DESCRIPTION
**Pull request by PennyScissors**
Add missing annotation and kubeVersion

### CODE REVIEW

**Review by aiyengar2 — APPROVED** [review]
LGTM

File: charts/streamsets/control-agent/2.0.101/Chart.yaml [hunk 1]
```diff
@@ -9,6 +10,7 @@ keywords:
 - streamsets
 - sdc
 - sch
+kubeVersion: 1.18 - 1.21
```
Comment (aiyengar2) [inline charts/streamsets/control-agent/2.0.101/Chart.yaml:13]: Based on Helm Docs this is valid: https://v3.helm.sh/docs/topics/charts/#the-kubeversion-field

`hyphen ranges for closed intervals, where 1.1 - 2.3.4 is equivalent to >= 1.1 <= 2.3.4.`

File: charts/streamsets/control-agent/2.0.101/Chart.yaml [hunk 2]
```diff
@@ -9,6 +10,7 @@ keywords:
 - streamsets
 - sdc
 - sch
+kubeVersion: 1.17 - 1.21
```
Comment (aiyengar2) [inline charts/streamsets/control-agent/2.0.101/Chart.yaml:13]: index.yaml should be modified https://github.com/rancher/partner-charts/blob/6f3b252e5c217636bcd575627fb7aca0b3ac4abd/index.yaml#L456-L477

Can you delete the above entry and run `make charts` to regenerate it?

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: charts/streamsets/control-agent/2.0.101/Chart.yaml (hunk 2 above)
Comment (PennyScissors) [reply charts/streamsets/control-agent/2.0.101/Chart.yaml:13]: Done

**Review by aiyengar2 — APPROVED** [review]
LGTM

**Pull request merged**